rasa/nlu/tokenizers/convert_tokenizer.py

```diff
@@ -37,6 +37,8 @@ def _load_tokenizer_params(self):
 
         model_url = "http://models.poly-ai.com/convert/v1/model.tar.gz"
 
+        # required to take care of cases when other files are already 
+        # stored in the default TFHUB_CACHE_DIR
```

Comment:
so we change default tfhub cache accumulating it? can we remove the cache instead

Reply:
no, we just direct the cache download to a specified directory so that TFHUB does not halucinate that it has already downloaded something;
As I understand it, the default, e.g., in my case, the default tfhub cache directory was used for something else before, so, if we remove the stuff in cache, we could remove other (non tf hub related) stuff the user has there.